html/user/edit_email_form.php

```diff
@@ -31,20 +31,24 @@
     $email_text = $user->email_addr;
 }
 
-form_start(secure_url_base()."/edit_email_action.php", "post");
-form_input_text(
-    tra("New email address").
-    "<br><p class=\"text-muted\">".tra("Must be a valid address of the form 'name@domain'")."</p>",
-    "email_addr", $email_text
-);
-
-// we need the password here not for verification,
-// but because we store it salted with email address,
-// which is about to change.
-
-form_input_text(tra("Password"), "passwd", "", "password");
-form_submit(tra("Change email address"));
-form_end();
+if ($user->email_addr_change_time + 604800 > time()) {
+    echo tra("Email address was changed within the past 7 days. Please look for an email to $user->previous_email_addr to verify this change.");
```

Comment:
The word "verify" feels wrong to me. You can't verify the change but revert it with the email sent to the previous email address.